frontend/src/components/Auth/Auth.js

```diff
@@ -40,7 +40,6 @@ const Auth = (props) => {
         isLoading,
     } = props;
     const [isResponse, setIsResponse] = useState(false);
-    const { t } = useTranslation('common');
```

Comment:
why you change hook useTranslation to i18n?

Reply:
Because we can use instance `i18n` and this object has a method **`t`** that does exactly what i useTranslation